### PR TITLE
Drop list for user

### PR DESCRIPTION
**Pull request by aydamacink**
The feature is ready and Tam's feature of making the drop's clickable is working. However, I passed the key  `id`for the value `drop.id`It should work well as soon as this is fixed :)

Also pls check the wording for the view

### CODE REVIEW

**Comment by aydamacink** [comment]
I need to check why the build is failing also

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  helper_method :current_user, :client
```
Comment (berlintam) [inline app/controllers/application_controller.rb:4]: very cool, I've never seen this before!
you use current_user in the user_drops/index, right? Where do you use client?

File: app/views/user_drops/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+  <div class="container-fluid">
+    <h1> Hello, <%= current_user.username %>! Here you can see the drops that you have added: </h1>
```
Comment (berlintam) [inline app/views/user_drops/index.html.erb]: Maybe something a bit simpler, like "Hello (user_name), here are your drops:"

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -19,6 +19,11 @@ def current_user
     @current_user ||= client.get('/me') if client.present?
   end
 
+  def require_current_user
```
Comment (berlintam) [inline app/helpers/application_helper.rb:22]: 👍 !

**Comment by berlintam** [comment]
This is really great! good job!
I just suggest to change the wording & then we can merge.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (aydamacink) [reply app/controllers/application_controller.rb:4]: current_user needs client so that's why it is also defined

**Comment by berlintam** [comment]
cool beans

**Pull request merged**